AboutDoc/Documentation_About.rst

```diff
@@ -0,0 +1,250 @@
+.. Copyright (c) 2020-2020 OpenShot Studios, LLC
+ (http://www.openshotstudios.com).
+ This file is part of OpenShot Video Editor (http://www.openshot.org),
+ an open-source project dedicated to delivering high quality video editing and animation solutions to the world.
+
+.. OpenShot Video Editor is free software:
+ you can redistribute it and/or modify it under the terms of the GNU General Public License as published by  the Free Software Foundation, 
+ either version 3 of the License,
+ or (at your option) any later version.
+
+.. OpenShot Video Editor is distributed in the hope that it will be useful,
+ but WITHOUT ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
+ See the GNU General Public License for more details.
+
+.. You should have received a copy of the GNU General Public License
+
+.. _Documentation_ref:
+
+About the documentation
+=======================
+
+The source files for the manual are all found in the project repository, `(the doc/ directory) <https://github.com/OpenShot/openshot-qt/tree/develop/doc>`_ 
+
+The documentation is written in reStructured Text, or ReST. 
+This is a plain text format encoded in UTF-8.
+It contains special syntax so formatting can be applied by third-party tools.
+The tool used by Openshot is Sphinx to create both the online HTML and the offline manual.
+
+You can suggest improvements or submit small changes for our documentation on our Github here: 
+https://github.com/OpenShot/openshot-qt/issues/2989
+
+Or in *this* reddit thread. 
+
+.. TODO: Reddit thread to be made, bookmarked?, add hyperlink 
+
+The preferred method for submitting large edits would be via GitHub Pull Request. 
+But we can make accommodations for anyone who would like to contribute but is not familiar with version-control systems like Git.
+
+License
+-------
+We use the GPL-3 license (see above) for documentation, see the header.
+This is for simplicity because it is the same license as the project code.
+And because the documentation gets parsed in other tools before it reaches its final form.
+
+Github
+------
+In the issue tracker, subjects that contain explanations that should probably be included in the documentation can be labeled ** `docs. <https://github.com/OpenShot/openshot-qt/labels/docs>`_ **
```

Comment:
According to [the docs](https://www.sphinx-doc.org/en/master/usage/restructuredtext/basics.html#gotchas) nested inline markup is simply not possible. There's no way to bold a hyperlink, or anything of that nature.
```suggestion
In the issue tracker, subjects that contain explanations that should probably be included in the documentation can be labeled `docs <https://github.com/OpenShot/openshot-qt/labels/docs>`_\ .
```